hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAUtils.java

```diff
@@ -89,7 +90,8 @@ private SCMHAUtils() {
   // Check if SCM HA is enabled.
   public static boolean isSCMHAEnabled(ConfigurationSource conf) {
     return conf.getBoolean(ScmConfigKeys.OZONE_SCM_HA_ENABLE_KEY,
-        ScmConfigKeys.OZONE_SCM_HA_ENABLE_DEFAULT);
+        DefaultConfigManager.getValue(ScmConfigKeys.OZONE_SCM_HA_ENABLE_KEY,
```

Comment:
This might be a good place to add a block comment explaining in which cases Ratis will or will not be used by default based on user configuration and cluster state.

Reply:
Have added a comment block where the configs are being validated.